### PR TITLE
Support to execute subscribers in parallel

### PR DESCRIPTION
**Pull request by thakurajayL**
We had support to execute all the profiles in parallel. Now this
pull request plans to extend support to execute multiple UEs in
parallel.

### CODE REVIEW

File: profile/profile.go [hunk 1]
```diff
@@ -81,47 +81,69 @@ func ExecuteProfile(profile *profctx.Profile, summaryChan chan common.InterfaceM
...
-			case common.PROFILE_PASS_EVENT:
-				profile.Log.Infoln("Result: PASS, imsi:", msg.Supi)
-				summary.UePassedCount++
-			case common.PROFILE_FAIL_EVENT:
-				err := fmt.Errorf("imsi:%v, procedure:%v, error:%v", msg.Supi, msg.Proc, msg.Error)
-				profile.Log.Infoln("Result: FAIL,", err)
+		wg.Add(1)
+		go func(simUe *simuectx.SimUe) {
+			defer wg.Done()
+			err := ExecuteSimUe(profile, simUe, imsiStr)
```
Comment (badhrinathpa) [inline profile/profile.go:100]: error might be overwritten by parallel procedures ?

**Review by badhrinathpa — APPROVED** [review]
+1

File: profile/profile.go (hunk 1 above)
Comment (thakurajayL) [reply profile/profile.go:100]: err has local scope within go routine, so we are safe.

**Review by badhrinathpa — APPROVED** [review]
+1

**Pull request merged**